Makefile.ci

```diff
@@ -85,6 +86,8 @@ ci-metacoq: ci-equations
 
 ci-vst: ci-flocq
 
+ci-compcert: ci-menhir
```

Comment:
It should also depend on `ci-flocq` shouldn't it?

Reply:
done